### PR TITLE
Add scheduled scanning to the templates for GitLab and GitHub

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

File: docs/install-gitlab.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ frogbot-scan:
       variables:
         FROGBOT_CMD: "create-fix-pull-requests"
         JF_GIT_BASE_BRANCH: $CI_COMMIT_BRANCH
+    - if: $CI_PIPELINE_SOURCE == "schedule"
```
Comment (yahavi) [inline docs/install-gitlab.md:29]: Looks repetition of push event.
Can we use "or"?
```yml
- if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH || $CI_PIPELINE_SOURCE == "schedule"
```

File: docs/templates/github-actions/scan-and-fix/frogbot-scan-and-fix-go.yml [hunk 2]
```diff
@@ -1,11 +1,14 @@
 name: "Frogbot Scan and Fix"
 on:
   push:
-    # Creating fix pull requests will be triggered by any push to one of the these branches.
+    # Creating fix pull requests will be triggered by any push to one of the these branches.q
```
Comment (yahavi) [inline docs/templates/github-actions/scan-and-fix/frogbot-scan-and-fix-go.yml:4]:
```suggestion
    # Creating fix pull requests will be triggered by any push to one of these branches.
```

Let's remove the "the" in all other templates

**Review by yahavi — CHANGES_REQUESTED** [review]

File: docs/install-gitlab.md [hunk 3]
```diff
@@ -22,7 +22,8 @@ frogbot-scan:
       # Creating fix pull requests will be triggered by any push to the default branch.
       # You can change it to any other branch you want, for example:
       # if: $CI_COMMIT_BRANCH == "dev"
-    - if: $CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH
+      # For scheduled pipelines, creating fix pull requests will be triggered as well.
```
Comment (eyalbe4) [inline docs/install-gitlab.md:25]: I suggest removing this line.

File: docs/install-gitlab.md [hunk 4]
```diff
@@ -22,7 +22,8 @@ frogbot-scan:
       # Creating fix pull requests will be triggered by any push to the default branch.
```
Comment (eyalbe4) [inline docs/install-gitlab.md:22]: Creating fix pull requests will be triggered by any push to the default branch.
-->
Repository scanning is triggered by any push to the default branch.

File: docs/install-gitlab.md [hunk 5]
```diff
@@ -22,7 +22,8 @@ frogbot-scan:
...
       # You can change it to any other branch you want, for example:
```
Comment (eyalbe4) [inline docs/install-gitlab.md:23]: You can change it to any other branch you want, for example:
-->
If you;d like a different branch to be scanned, replace $CI_DEFAULT_BRANCH in the line below with the name of branch, wrapped with quotes (").

File: docs/install-gitlab.md [hunk 6]
```diff
@@ -22,7 +22,8 @@ frogbot-scan:
...
       # if: $CI_COMMIT_BRANCH == "dev"
```
Comment (eyalbe4) [inline docs/install-gitlab.md:24]: Let's remove this line.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Pull request merged**